Configuration/Geometry/cmsExtendedGeometry2026D98XML_cfi.py

```diff
@@ -0,0 +1,130 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
why is this file needed? 
seems a copy of `Geometry/CMSCommonData/python/cmsExtendedGeometry2026D98XML_cfi.py`